README.md

```diff
@@ -45,6 +45,7 @@ python3 downwash.py
 cd gym_pybullet_drones/examples/
 python learn.py # task: single drone hover at z == 1.0
 python learn.py --multiagent true # task: 2-drone hover at z == 1.2 and 0.7
+python play.py --model_path results/FOLDER_NAME/best_model.zip # play and visualize learned policy after training
```

Comment:
Thanks @fanshi14 , I will test this PR later today and see if we can add a simple CLI way to get the folder name so that the readme can be truly copy-and-paste